web/src/components/molecules/Asset/Asset/AssetBody/preview-toolbar.tsx

```diff
@@ -0,0 +1,48 @@
+import Button from "@reearth-cms/components/atoms/Button";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import PreviewModal from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/preview-modal";
+import { PreviewType } from "@reearth-cms/components/molecules/Asset/Asset/AssetBody/preview-type-select";
+
+type PreviewToolbar = {
+  url: string;
+  selectedPreviewType: string;
+  isModalVisible: boolean;
+  handleCodeSourceClick: () => void;
+  handleRenderClick: () => void;
+  handleFullScreen: () => void;
+  handleModalCancel: () => void;
+};
+
+const PreviewToolbar: React.FC<PreviewToolbar> = ({
+  url,
+  selectedPreviewType,
+  handleCodeSourceClick,
+  handleRenderClick,
+  handleFullScreen,
+  isModalVisible,
+  handleModalCancel,
+}) => {
+  return (
+    <>
+      {selectedPreviewType === PreviewType.SVG && (
+        <>
+          <Button onClick={handleCodeSourceClick}>Code Source</Button>
```

Comment:
Source code?

Reply:
I know that source code is the correct one, but I am writing according to the design in the figma file:
<img width="1012" alt="Screen Shot 0004-07-28 at 5 43 47 PM" src="https://user-images.githubusercontent.com/35835415/181463027-fe12459c-fb2e-465c-8087-a2b039539ece.png">

Reply:
Renamed to Source code.